src/execution_plan/ops/op.c

```diff
@@ -238,6 +238,20 @@ bool OpBase_IsWriter
 	return op->writer;
 }
 
+// indicates if the operation is an eager operation
+bool OpBase_IsEager
```

Comment:
there is already one like this called ExecutionPlan_isEager

Reply:
A bit confusing but `ExecutionPlan_isEager` checks to see if the Execution-Plan spanned from the given root contains an eager operation while `OpBase_IsEager` checks if the given operation is an eager op.